content/riak/kv/2.2.0/using/performance/open-files-limit.md

```diff
@@ -15,326 +15,278 @@ aliases:
+two asterisks (`*`) in the examples with `riak`.
 {{% /note %}}
 
 
 ## Solaris
 
-In Solaris 8, there is a default limit of 1024 file descriptors per
-process. In Solaris 9, the default limit was raised to 65536. To
-increase the per-process limit on Solaris, add the following line to
-`/etc/system`:
+To increase the open file limit on Solaris, add the following line to the /etc/system file:
 
-```config
+```/etc/system
 set rlim_fd_max=200000
 ```
 
-[Reference](http://blogs.oracle.com/elving/entry/too_many_open_files)
+[Reference][blog oracle]
+
+## Mac OSX El Capitan
+
+Start by checking the current open file limit values with:
+
+```bash
+launchctl limit maxfiles
+```
+
+The response should looking something like this:
+
+```bash
+maxfiles    65536          65536
+```
+
+The first column is the soft limit and the last column is the hard limit.
 
-## Mac OS X
+To change the open files limits on Mac OS X Yosemite, perform the following steps:
 
-To check the current limits on your Mac OS X system, run:
+1\. Add the following line to your .bash_profile or analogous file:
+
+```bash
+ulimit -n 65536 200000
+```
+
+2\. Save and close the file. Next open /etc/sysctl.conf (or create it if it doesn't already exist) and add the following settings:
+
+```/etc/sysctl.conf
+kern.maxfiles=200000
+kern.maxfilesperproc=200000
+```
+
+4\. Restart your computer and enter `ulimit -n` into your terminal. If your system is configured correctly, you should see that `maxfiles` has been set to 200000.
+
+
+## Mac OSX Yosemite
+
+Start by checking the current open file limit values with:
 
 ```bash
 launchctl limit maxfiles
 ```
 
-The last two columns are the soft and hard limits, respectively.
-
-### Adjusting Open File Limits in Yosemite
-
-To adjust open files limits on a system-wide basis in Mac OS X Yosemite,
-you must create two configuration files. The first is a property list
-(aka plist) file in `/Library/LaunchDaemons/limit.maxfiles.plist` that
-contains the following XML configuration:
-
-```xml
-<?xml version="1.0" encoding="UTF-8"?>
-<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
-  <plist version="1.0">
-    <dict>
-      <key>Label</key>
-        <string>limit.maxfiles</string>
-      <key>ProgramArguments</key>
-        <array>
-          <string>launchctl</string>
-          <string>limit</string>
-          <string>maxfiles</string>
-          <string>200000</string>
-          <string>200000</string>
-        </array>
-      <key>RunAtLoad</key>
-        <true/>
-      <key>ServiceIPC</key>
-        <false/>
-    </dict>
-  </plist>
+The response should looking something like this:
+
+```bash
+maxfiles    65536          65536
+```
+
+The first column is the soft limit and the last column is the hard limit.
+
+To change the open files limits on Mac OS X Yosemite, perform these steps:
+
+1\. Add the following line to your .bash_profile or analogous file:
+
+```bash
+ulimit -n 65536 200000
```

Comment:
Did these instructions get added twice or were they supposed to be different Mac versions?